recipes/last/meta.yaml

```diff
@@ -1,34 +1,33 @@
-about:
-    home: 'http://last.cbrc.jp/'
-    license: GPLv3
-    summary: "LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads)."
 package:
   name: last
-  version: '847'
+  version: "852"
+
 source:
-  fn: last-847.zip
-  md5: 45c49cd2ce271ec0ef9cc7119feb2c9f 
-  url: http://last.cbrc.jp/last-847.zip
-  patches:
-    - maf-convert.23patch
+  fn: last-852.zip
+  url: http://last.cbrc.jp/last-852.zip
+  md5: 96b8fdadab2b0ed77f8e55ce4c91687f
 
 build:
-  number: 0
-  skip: False
+  number: 1
+  skip: True # [py3k]
```

Comment:
Is there any reason to skip py3k? Is it not supported, if so are the older version also not supported for py3k?